tools/logger/convert.c

```diff
@@ -726,7 +726,7 @@ static int dump_ldc_info(struct convert_config *config,
 		  ((uintptr_t)uids_dict + uids_dict->data_offset);
 
 	while (remaining > 0) {
-		name = format_uid_raw(uid_ptr, 0, 0);
+		name = format_uid_raw(&uid_ptr[cnt], 0, 0);
 		uid_addr = (uintptr_t)&uid_ptr[cnt] - (uintptr_t)uids_dict -
 			    uids_dict->data_offset + uids_dict->base_address;
 		fprintf(out_fd, "\t0x%lX  %s\n", uid_addr, name);
```

Comment:
Can you describe a little bit the problem solved? What was happening before this patch and with this patch.

This patch looks good per se, but the commit is not relevant. What you just said in the commit message can be noticed from the code.

The commit message should really explain more WHY the patch is needed and less WHAT the patch is doing. What the patch is doing should be obvious from the code.

Reply:
I added output before and after this change to commit message.